File: multipleElementsExceptOwnPoitionElement.py
# import math

class Solution(object):
  def multipleElementsExceptOwnPoitionElement(self, list):
    if not (len(list)):
      return list
        
    total=1
    for i in list:
  	   total*=i
  	
    newList=[]
    for j in list:
  	  newList.append(total//j)
    
    return newList

obj=Solution()
print(obj.multipleElementsExceptOwnPoitionElement([1,2,3,4]))

# print(singleDigit(100))

File: multipleElementsExceptOwnPoitionElement.js
function multipleElementsExceptOwnPoitionElement(list) {
  if(!list.length){
    return list
  }
  let total = 1;
  for (let i=0;i<list.length;i++){
    total *= list[i]
  }
  
  let newList = [];
  for (let j=0;j<list.length;j++){
    newList.push(total/list[j])
  }
  return newList
}

multipleElementsExceptOwnPoitionElement([1,2,3,4])